app/models/automated_checks/dqt_report_csv.rb

```diff
@@ -0,0 +1,50 @@
+require "csv"
+
+module AutomatedChecks
+  class DQTReportCsv
```

Comment:
Also have you seen https://ruby-doc.org/stdlib-2.6.3/libdoc/csv/rdoc/CSV.html#HeaderConverters? It might be a good idea to use that functionality to covert the wieldly named headers to headers that are more easy to work with elsewhere in the codebase. e.g. we'd be able to call `row.qts_year` instead of `row["dfeta qtsdate"]`.

Reply:
I originally had a header converter in this class, however when it came to replaying the missing columns back to the user we felt it was quite confusing for the user when giving the converted headers in the error message. They would look at the CSV and none of the headings would match.

I'll have a think, to see if we can convert the headings but preserve the actual heading names for the error message.